.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+<!-- A clear and concise description of what the bug is. -->
+
+## To Reproduce
+<!--
+Please provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+
+>Your code examples should be…
+>   …Minimal – Use as little code as possible that still produces the same problem
+>   …Complete – Provide all parts someone else needs to reproduce your problem in the issue itself
+>   …Reproducible – Test the code you're about to provide to make sure it reproduces the problem
+-->
+
+## Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Actual behavior
+<!-- A clear and concise description of what happened instead. -->
+
+## Screenshots
```

Comment:
Btw. is it really likely screenshots will be necessary for Spock issues?

Reply:
Does it have to be likely in order to be possible? Maybe someone wants to show a screenshot of an HTML report or what a test result looks like in the IDE, just in order to illustrate something. Something like https://github.com/spockframework/spock/issues/1268, maybe.

Having said that, I agree with the assumption contained in your rhetorical question, that it should be unlikely.

Reply:
We've got them from time to time.

Reply:
It was not a rhetorical question at all.
Rhetorical questions don't expect an answer.
This question did.

If someone want to show a screenshot he can always do so, whether there is a section in the template or not.
It would even fit under "Additional context" if in doubt.
I just questioned whether we need a dedicated section in the template if in 99% of the cases there will be no screenshots to show.
Of course we can leave it in, I just gave it to consider. ;-)